pkg/apis/messaging/v1alpha1/subscription_types.go

```diff
@@ -191,10 +175,10 @@ type SubscriptionStatus struct {
 // Subscription.
 type SubscriptionStatusPhysicalSubscription struct {
 	// SubscriberURI is the fully resolved URI for spec.subscriber.
-	SubscriberURI string `json:"subscriberURI,omitempty"`
```

Comment:
should this be a pointer type?

Reply:
yeah, I mistook the lack of pointer in the spec that it shouldn't have been a pointer :) my bad :)